### PR TITLE
treewide: remove unused/redundant arguments, format workspace module, fix whitespace, remove `with lib;`

### PR DESCRIPTION
**Pull request by SigmaSquadron**
Various code refactoring changes. We may want to make use of Nixpkgs' EditorConfig settings/CI for this repo as well.

One of the commits is marked as `NEEDS DISCUSSION`. Replacing declared, yet unused arguments with `...` is frowned upon by some, but others prefer having no unused args. Let me know if that commit should be deleted.

### CODE REVIEW

File: modules/input.nix [hunk 1]
```diff
@@ -433,77 +431,77 @@ in
...
-    options = mkOption {
-      type = with types; nullOr (listOf str);
+    options = lib.mkOption {
+      type = with lib.types; nullOr (listOf str);
       default = null;
       example = [
         "altwin:meta_alt"
         "caps:shift"
-        "custom:types"
+        "custom:lib.types"
```
Comment (HeitorAugustoLN) [inline modules/input.nix]: This shouldn't change.
```suggestion
        "custom:types"
```

**Review by HeitorAugustoLN — CHANGES_REQUESTED** [review]

**Comment by HeitorAugustoLN** [comment]
Everything else looks good. I really liked the change from the unused args to `...`

File: modules/input.nix (hunk 1 above)
Comment (SigmaSquadron) [reply modules/input.nix]: Whoops! `rg` was a bit too eager there. Resolved.

**Comment by SigmaSquadron** [comment]
Changes were addressed, the commits were renamed per convention, and there's a new commit to modernise/normalise the URLs in the documentation/code commits.

I wasn't exactly sure how to handle your merge commit though. I removed it and rebased manually.

**Review by HeitorAugustoLN — APPROVED** [review]

**Comment by HeitorAugustoLN** [comment]
Thanks for your contribution again :)

**Pull request merged**